srt-protocol/src/protocol/pending_connection/hsv5.rs

```diff
@@ -33,13 +32,36 @@ pub fn gen_hsv5_response(
         }
     };
 
+    let mut acceptor = AllowAllStreamAcceptor::default();
     let mut accept_params = match acceptor.accept(incoming.sid.as_deref(), from) {
```

Comment:
This can be simplified--AllowAllStreamAcceptor always returns Ok(AcceptParameters::default())

Reply:
right, I thought I had gotten rid of all of these